aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/services/JobsServiceImpl.java

```diff
@@ -128,9 +135,23 @@ public String startAddVersionJob(JobRequestBuilder builder) throws JobServiceExc
         return startJob(builder);
     }
 
+    private CreateJobsRequest filterModuleGenerationType(CreateJobsRequest jobRequest) {
+        String moduleTypeKey = "moduleGenerationType";
+        String moduleTypeValue = jobRequest.getJobParameters().get(moduleTypeKey);
+        ModuleGenerationType moduleType = ModuleGenerationType.fromString(moduleTypeValue);
+        if (EnumSet.of(JobType.ANALYZE, JobType.CLONE_VERSION).contains(jobRequest.getJobType()) && moduleType != null && moduleType == ModuleGenerationType.ONE_PER_TECHNO) {
```

Comment:
You can simply have `moduleType == ONE_PER_TECHNO` since if moduleType is null, then the comparison will be false (null is different from ONE_PER_TECHNO)